code/game/objects/items/holy_weapons.dm

```diff
@@ -532,6 +532,10 @@
 	attack_verb_simple = list("enlighten", "redpill")
 	menu_description = "A sharp fedora dealing a very high amount of throw damage, but none of melee. Fits in pockets. Can be worn on the head, obviously."
 
+/obj/item/nullrod/fedora/suicide_act(mob/user)
+	user.visible_message(span_suicide("[user] is killing [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to get further from god!"))
```

Comment:
```suggestion
	user.visible_message(span_suicide("[user] is killing [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to get further from God!"))
```

Reply:
the normal null rod suicide also doesn't capitalize "god"

Reply:
eh

Reply:
"God" is a proper noun. As in, "He believes in God." or "He prayed to God."
"god" is not a proper noun.  As in, "He believes in a god." or "He prayed to some god or another."
As there is no article in this sentence, I would infer that "God" is being used as a proper noun and would require a capital.

Reply:
Wouldn't it be better to just use whatever deity name the chap set and sidestep the whole issue?

Reply:
Yeah but if someones running the atheism gimmick they'd be getting further from anti-god or whatever

Reply:
I believe "gods" would be correct, God by big g implies monotheism, while atheist doesn't believe in any god, from monotheist and politheist religions alike. Also, "god" is a proper noun while talking about politheistic religions, or many at once at least. Zeus was a god of thunder, christian god shares some traits with Aztec gods, etc. Not to get into the fact that capitalizing type of creature (god) of something that has a name (Yahwe, in christianity's case) is only to show the respect to it, which atheist doesn't really have reason to have, but whatever.

Reply:
![image](https://user-images.githubusercontent.com/40974010/141865086-cdacc6c5-c15f-4744-8349-c9da825a8941.png)